src/services/Jwts.php

```diff
@@ -155,6 +155,14 @@ public function getTokenPayload(string $token)
 
             // Do we have any data?
             if (isset($decodedJwt->data) && !empty($decodedJwt->data)) {
+                // Parse the payload to an object, if it's a valid json string.
+                if (is_string($decodedJwt->data)) {
+                    $jsonPayload = json_decode($decodedJwt->data);
+                    if (!is_null($jsonPayload)) {
```

Comment:
Code's okay, just could be shortened. Please change to: `return $jsonPayload ? $jsonPayload : null;`